blocks/sidebar/sidebar.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Fragment Block
+ * Include content from one Helix page in another.
+ * https://www.hlx.live/developer/block-collection/fragment
+ */
```

Comment:
Does this need to be called `sidebar` or could we simply use the `fragment` block name and have it generic, so it can be reused in other contexts?